### DISCUSSION TITLE
Docker image help

### DISCUSSION THREAD

**Discussion by orrinwitt** [bitfocus/companion#2426] [General]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

I have found the mention of and link to a docker image for Companion, but could we get more information about installing and running it? 
I'd like to run Companion from my server (TrueNAS SCALE), and interact with it via Companion Satellite on a Raspberry Pi.
However, this is proving to be WAY above my level of expertise.
I've submitted a request for a proper "app" to be developed for TrueNAS SCALE  on Truecharts, but I don't know if that's going anywhere.

**Comment by Julusian** [comment]
The images can be pulled from https://github.com/bitfocus/companion/pkgs/container/companion%2Fcompanion

There isn't much documentation on them currently, but the basics for 3.0 (beta) is:
* You must mount some storage to `/companion` for the config to be stored in
* If you want to load in some custom versions of modules, this can be done at the path `/app/module-local-dev`, where they will be detected and run.
* You will need to expose port 8000 for the webui, and you may with to expose other ports too, depending on what you need. For satellite, you will want to expose 16622

I think that 2.4.2 is the same, but I am not 100% certain on that.
